src/styles/components/_about-us-introduction.scss

```
@@ -0,0 +1,99 @@
+.about-us-introduction {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(400px);
+  }
+
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__title {
+    width: 58%;
+    height: 12.5%;
+    margin-block-start: 0%;
+    margin-block-end: 0%;
+
+    color: $dark;
+    -webkit-text-stroke: rem(1px) $dark;
```

```suggestion
    -webkit-text-stroke: 1px $dark;
```
This one is waste of ``rem`` function call. If only 1 px don't need a ``rem`` function to cover. 😀